spring-rabbit/src/main/java/org/springframework/amqp/rabbit/listener/SimpleMessageListenerContainer.java

```diff
@@ -121,6 +124,8 @@ public class SimpleMessageListenerContainer extends AbstractMessageListenerConta
 
 	private long receiveTimeout = DEFAULT_RECEIVE_TIMEOUT;
 
+	private long batchReceiveTimeout = DEFAULT_BATCH_RECEIVE_TIMEOUT;
```

Comment:
I don't think we can assume some default.
This has to be an `opt-in` property.
Otherwise it is a breaking change for the current patch version.
Therefore I suggest to have a default as it is right now: no any batch timeouts.